app/js/stores/QuestionStore.js

```diff
@@ -114,6 +122,10 @@ QuestionStore.dispatchToken = AppDispatcher.register((action) => {
       loadTopQuestions(action.data.questions);
       QuestionStore.emitChange();
       break;
+    case ZhishiConstants.RECEIVE_USER_QUESTIONS:
+      loadUserQuestions(action.data.questions);
+      QuestionStore.emitChange();
+      break;
```

Comment:
What if this response comes with `action.data` being null? To avoid an app break, you may want to put a conditional check. Something like:
```
    case ZhishiConstants.RECEIVE_USER_QUESTIONS:
       if (action.data) {
          loadUserQuestions(action.data.questions);
          QuestionStore.emitChange();
       }
       break;
```